lib/headers.sh

```diff
@@ -19,3 +19,5 @@ source "${TMUX_POWERLINE_DIR_LIB}/formatting.sh"
 source "${TMUX_POWERLINE_DIR_LIB}/muting.sh"
 source "${TMUX_POWERLINE_DIR_LIB}/powerline.sh"
 source "${TMUX_POWERLINE_DIR_LIB}/rcfile.sh"
+
```

Comment:
Segments should not be sourced here. Please remove and verify that the segment still works :).